stream/ResponsePartIterator.java

```diff
@@ -91,8 +91,7 @@ public boolean hasNext() {
 
     @Override
     public TransactionProto.Transaction.ResPart next() {
-        if (stream.getError().isPresent()) throw TypeDBClientException.of(stream.getError().get());
```

Comment:
the error, if there is one, will be throw via the `fetchAndCheck()` method which will read the error from it's internal queue